microsetta_public_api/repo/_alpha_repo.py

```diff
@@ -19,17 +95,27 @@ def get_alpha_diversity(self, sample_ids, metric):
             Sets the name of the series to `metric`.
 
         """
+        # this could raise an UnknownMetric or ConfigurationError
+        alpha_series = self._load_resource(metric)
+        # TODO the following could throw KeyErrors if a sample id is not in
+        #  the index. Right now, the API checks for missing ID's before
+        #  calling this.
+        if isinstance(sample_ids, str):
+            return alpha_series.loc[[sample_ids]]
```

Comment:
doesn't this return the same as `alpha_series.loc[sample_ids]` when `sample_ids` is a `str`?

Reply:
No.
```
$ python
Python 3.6.10 |Anaconda, Inc.| (default, Mar 25 2020, 18:53:43) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pandas as pd
>>> foo = pd.Series({'bar': 'baz', 'qux': 'quux'})
>>> foo.loc['bar']
'baz'
>>> foo.loc[['bar']]
bar    baz
dtype: object

```

Reply:
oh i see, cool, thanks